openapi/datasource.yaml

```diff
@@ -212,6 +212,7 @@ components:
             - AWSIAMRoleWithUsername
             - GCPBasic
             - OAuth
+            - ClientIdSecret
```

Comment:
this might need you to regenerate the datasource client as well

Reply:
Already regenerated. There is no difference cause this enum was manually added before in the generated code.